docs/Q-and-A.md

```diff
@@ -0,0 +1,649 @@
+
+##  *Q: Where can we find the code and how could a coder get started?
+The homepage on github [README.md](../README.md) pretty much sums up all the possibilities to download the available code and how developers can engage in the development process currently. We welcome contributions.
+
+##  **Q: What are the external dependencies of Sidetree?
+_I hear ya: "Sidetree is able to do all this **without requiring** trusted intermediaries, centralized authorities, special protocol tokens, or secondary consensus mechanisms, while preserving the core attributes of decentralization and immutability of the underlying ledger systems it is implemented on." But what does Sidetree require instead?_
+
+Sidetree is dependent of:
+1. A content addressable storage layer (e.g., IPFS)
+2. Reasonable fees and confirmation-times on layer 1
+3. Consensus, transaction validation and governance in general of the layer 1 blockchain it settles on.
+4. Ongoing development and blockchain maintenance for safety of the records now and in the future.
+5. {TBW prio 2}
+_(@henkvancann)_
+
+##  **Q: In what can we find the efficiency booster that Sidetree promises to be?
+Sidetree nodes that participate in writing operations into the overlay network do so by **anchoring** Content-Addressable Storage (`CAS`) (e.g. IPFS) references to **aggregated** bundles of operations in an underlying ledger.
+
+##  ***Q: "Process operations in accordance with the protocol deterministic ruleset." What does this mean?
+Read this [section](https://identity.foundation/sidetree/spec/#late-publishing) that illustrates a DID owner, Alice, creating forks by creating and anchoring operations in the past that she does not expose to the network. Known as _Late Publishing_ of a DID operation.\
+This type of processing guarantees only one `fork` of a DID’s state history can ever be valid.
+
+##  *Q: What would you see as the main drawback of Sidetree?
+There aren't many. Sidetree has Self-Certifying Identifiers, however they are non-transferable. {Right?}
+_(@henkvancann)_
+
+##  *Q: How can it be one solution, fit for more DIDs? 
+DID Methods based on the Sidetree protocol all share the same identifier format. The unique identifier segment of a Sidetree-based DID, known as the DID Suffix, is derived by using the Hashing Process to generate a hash value from the canonicalized Create Operation Suffix Data Object. The DID Suffix is cryptographically bound to the initial PKI state of the DID, which means Sidetree DIDs are also self-certified.
+
+## *Q: Where you would need something quite different than Sidetree to solve a similar problem?
+In a private situation with no self sovereignty. A Certification Authority in combination with a federated setup could do, like it has done the past decades. But in that SSI become Self Service Identity at best, because the Identifiers are owned and controlled by a third party.
+_(@henkvancann)_
+
+## **Q: How does Sidetree scale
+This image speaks a thousand words:\
```

Comment:
nit: formatting of the image looks weird

Reply:
I agree, but it is what it is :)